packages/nextra-theme-docs/src/utils/get-git-issue-url.ts

```diff
@@ -1,4 +1,4 @@
-import parseGitUrl from 'parse-git-url'
+import parseGitUrl from 'git-url-parse'
```

Comment:
can we rename it to `gitUrlParse` to make consistency with his docs? I don't like `GitUrlParse` because I prefer names with uppercase only classes